src/Raven.Studio/package.json

```diff
@@ -24,6 +24,7 @@
         "@hookform/devtools": "^4.3.1",
         "@storybook/addon-a11y": "^8.4.7",
         "@storybook/addon-actions": "^8.4.7",
+        "@storybook/addon-designs": "^8.0.4",
```

Comment:
can you match version with storybook? ^8.4.7

Reply:
@ml054 updated to the [latest](https://github.com/storybookjs/addon-designs/releases) (8.2.0)